### PR TITLE
VST QSG update to add subplotID info to text and figure

### PR DESCRIPTION
**Pull request by schafnoir**
Update also adds minor text modifications throughout document to improve clarity and accuracy.

### CODE REVIEW

File: DP1.10098.001/Standard.calculations.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
 
-To extrapolate from individual plants to the site level, adjustment should be made for the distribution of measured plants across the site as whole. The nlcdClass field in the vst_perplotperyear table indicates the National Land Cover Database class of each plot, and the vst_perplotperyear table also contains the results of the annual survey of candidate plots to determine the presence or absence of relevant plant types in each plot.
+To extrapolate from individual plants to the plot or site level, adjustment should be made for the plot-specific use of nested subplots (for smaller growth forms), and the distribution of measured plants across the site as a whole. The nlcdClass field in the vst_perplotperyear table indicates the National Land Cover Database class of each plot, and the vst_perplotperyear table also contains the results of the annual survey of candidate plots to determine the presence or absence of relevant plant types in each plot.
```
[the remark this answers is not in the document]
Comment (schafnoir) [reply DP1.10098.001/Standard.calculations.txt:3]: @cklunch  - yes, that's exactly correct. Within the same plotID, different nestedSubplotArea can be used for different growth forms (Shrubs/saplings, lianas, ferns, and all 'other'), and nestedSubplotArea can also vary from plot-to-plot for the same growth form. Thus, to calculate biomass, stem density, etc. at the plot scale one must use the nestedSubplotArea per growthForm per plot.

[the remark this answers is not in the document]
Comment (cklunch) [reply DP1.10098.001/Standard.calculations.txt:3]: @schafnoir OK thanks! Can you rephrase in the doc?

[the remark this answers is not in the document]
Comment (schafnoir) [reply DP1.10098.001/Standard.calculations.txt:3]: Sure thing! I should just be able to make another commit with updated text to my existing fork, correct?

[the remark this answers is not in the document]
Comment (cklunch) [reply DP1.10098.001/Standard.calculations.txt:3]: @schafnoir Yep!

[the remark this answers is not in the document]
Comment (schafnoir) [reply DP1.10098.001/Standard.calculations.txt]: @cklunch  - I just committed an update to the Standard Calculations text in my fork. Lemme know if this is clearer and we're on the right track!

**Comment by cklunch** [comment]
Looks good, thanks! I'll need to re-generate the QSG and then wait for a Portal sync, so this may not show up on Portal until Tuesday.
